build.gradle

```diff
@@ -156,30 +148,29 @@ task cucumberTestReport(type: TestReport) {
     reportOn cucumberTest
 }
 
-apply from: 'gradle/docker.gradle'
-apply from: 'gradle/sonar.gradle'
-//jhipster-needle-gradle-apply-from - JHipster will add additional gradle scripts to be applied here
 
-if (project.hasProperty('prod')) {
-    apply from: 'gradle/profile_prod.gradle'
-} else {
-    apply from: 'gradle/profile_dev.gradle'
+checkstyle {
+    toolVersion = "7.6.1"
```

Comment:
Why hardcode, it's better to use variable like `"{$checkstyle_tool_version}"` from gradle.properties

Reply:
done